examples/tinywl-new/rootsurfacecontainer.cpp

```diff
@@ -44,6 +44,11 @@ RootSurfaceContainer::RootSurfaceContainer(QQuickItem *parent)
     });
 
     connect(m_outputLayout, &WOutputLayout::notify_change, this, [this] {
+
+        // The default layout is horizontal to the right
+        if (m_currentOutput && m_outputLayout->implicitWidth())
+            m_currentOutput->outputItem()->setX(m_outputLayout->implicitWidth());
```

Comment:
Must update position for which output:

````
for (....)
    output->updatePositionFromOutputLayout();
````

Reply:
done